.github/workflows/terratest.yml

```diff
@@ -28,13 +28,14 @@ jobs:
         id: go
       - name: prepare test environment
         run: source ./test/prepare-test-environment.sh
-      - name: Login to Google Cloud
-        uses: google-github-actions/setup-gcloud@master
+      - name: gcloud-auth
+        uses: google-github-actions/auth@v0
+        credentials_json: ${{ env.GOOGLE_CREDENTIALS }}
+      - name: gcloud-setup
+        uses: google-github-actions/setup-gcloud@v0
         with:
-          version: "latest" # This is the default value anyways, just being explicit
+          version: "latest" # This is the default value, just being explicit
```

Comment:
In my opinion, the version should be specification because the latest version may not away compatible.

Reply:
As mentioned in their docs, they recommend going to the latest version: https://github.com/google-github-actions/setup-gcloud#versioning

The cloud-native world moves fast :)

Reply:
Thank you :)